### PR TITLE
feat(components-links-group): add links-group-button-widget

### PR DESCRIPTION
**Pull request by Kaskyi**

### CODE REVIEW

**Review by Sumragen — APPROVED** [review]

**Review by ibilous — APPROVED** [review]

File: packages/components/links-group-widget/links-group-button-widget.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    selector: 'xm-links-group-button-widget',
+    template: `
+        <div *ngIf="config?.list" class="mb-3">
+            <nav class="xm-button-group p-3 bg-surface"
+                 role="group">
+                <ng-container *ngFor="let item of config.list">
+                    <a [routerLink]="item.routerLink"
+                       *xmPermission="item.privileges"
+                       routerLinkActive="active"
+                       #rla="routerLinkActive"
```
Comment (avchugaev) [inline packages/components/links-group-widget/links-group-button-widget.ts:21]: It seems `rla` is not in use. Please remove it.

File: packages/components/links-group-widget/links-group-button-widget.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    styleUrls: ['./links-group-widget.component.scss'],
+})
+export class LinksGroupButtonWidget {
+    @Input()
+    public config: LinksGroupWidgetConfig;
+}
+
+@NgModule({
+    declarations: [LinksGroupButtonWidget],
+    entryComponents: [LinksGroupButtonWidget],
```
Comment (avchugaev) [inline packages/components/links-group-widget/links-group-button-widget.ts:41]: Is this mandatory? 🤔

File: packages/components/links-group-widget/links-group-widget.ts [hunk 3]
```diff
@@ -41,37 +42,15 @@ interface Config {
     `,
     styleUrls: ['./links-group-widget.component.scss'],
 })
-export class LinksGroupWidgetComponent implements OnInit, OnDestroy {
+export class LinksGroupWidget {
```
Comment (avchugaev) [inline packages/components/links-group-widget/links-group-widget.ts:45]: Please, do not remove "Component" suffix from components.

File: packages/components/links-group-widget/links-group-widget.ts [hunk 4]
```diff
@@ -82,5 +61,5 @@ export class LinksGroupWidgetComponent implements OnInit, OnDestroy {
     ],
 })
 export class LinksGroupWidgetModule {
-    public entry: Type<IWidget> = LinksGroupWidgetComponent;
+    public entry: Type<IWidget> = LinksGroupWidget;
```
Comment (avchugaev) [inline packages/components/links-group-widget/links-group-widget.ts:64]: Same here, leave suffix, please.

File: packages/components/links-group-widget/links-group-button-widget.ts (hunk 1 above)
Comment (Kaskyi) [reply packages/components/links-group-widget/links-group-button-widget.ts:21]: +

File: packages/components/links-group-widget/links-group-button-widget.ts (hunk 2 above)
Comment (Kaskyi) [reply packages/components/links-group-widget/links-group-button-widget.ts:41]: +

File: packages/components/links-group-widget/links-group-button-widget.ts (hunk 1 above)
Comment (avchugaev) [reply packages/components/links-group-widget/links-group-button-widget.ts:21]: Where is it being used then?

**Pull request merged**